<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SIMDISTestClass</class>
 <widget class="QMainWindow" name="SIMDISTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DlgMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="0">
     <widget class="QTextEdit" name="edtLog"/>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnInitialize">
        <property name="text">
         <string>Initialize</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="3" colspan="2">
       <widget class="QLineEdit" name="edtServerResponsePort"/>
      </item>
      <item row="0" column="8">
       <widget class="QPushButton" name="btnJoinLeaveHP">
        <property name="text">
         <string>Join</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>DCS Client 응답 요청</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>DCS Server</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>DCS Server 응답</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="edtServerResponseIP"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>DLPS 호스트 I/F 전술</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="5">
       <widget class="QLineEdit" name="edtClientRequestIP"/>
      </item>
      <item row="2" column="5">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="5">
       <widget class="QLineEdit" name="edtServerIP"/>
      </item>
      <item row="0" column="1" colspan="5">
       <widget class="QLineEdit" name="edtHPIP"/>
      </item>
      <item row="3" column="6" colspan="2">
       <widget class="QLineEdit" name="edtServerPort"/>
      </item>
      <item row="1" column="6" colspan="2">
       <widget class="QLineEdit" name="edtClientRequestPort"/>
      </item>
      <item row="0" column="6" colspan="2">
       <widget class="QLineEdit" name="edtHPPort"/>
      </item>
      <item row="2" column="6" colspan="2">
       <widget class="QLineEdit" name="edtServerName"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>SIMDIS IP/Port</string>
        </property>
       </widget>
      </item>
      <item row="4" column="8">
       <widget class="QPushButton" name="btnStartStopServer">
        <property name="text">
         <string>Start Server</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="5">
       <widget class="QLineEdit" name="edtSIMDISIP"/>
      </item>
      <item row="4" column="6" colspan="2">
       <widget class="QLineEdit" name="edtSIMDISPort"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>649</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>btnInitialize</tabstop>
  <tabstop>edtHPIP</tabstop>
  <tabstop>btnJoinLeaveHP</tabstop>
  <tabstop>edtClientRequestIP</tabstop>
  <tabstop>edtServerResponseIP</tabstop>
  <tabstop>edtServerIP</tabstop>
  <tabstop>edtLog</tabstop>
 </tabstops>
 <resources>
  <include location="SIMDISTest.qrc"/>
 </resources>
 <connections/>
</ui>
